spring-boot-project/spring-boot-test/src/test/java/org/springframework/boot/test/json/JacksonTesterIntegrationTests.java

```diff
@@ -81,6 +83,28 @@ public void typicalMapTest() throws Exception {
 				.isEqualTo(1);
 	}
 
+	@Test
+	public void stringLiteral() throws Exception {
```

Comment:
Same test than for Gson, just to confirm that there isn't a similar regression when using Jackson.